roles/ssh_hardening/templates/opensshd.conf.j2

```diff
@@ -73,14 +73,14 @@ LogLevel {{ sshd_log_level }}
 # -- see: (http://net-ssh.github.com/net-ssh/classes/Net/SSH/Transport/CipherFactory.html)
 #
 {# This outputs 'Ciphers <list-of-ciphers>' if ssh_ciphers is defined or '#Ciphers' if ssh_ciphers is undefined -#}
-{{ 'Ciphers ' ~ ssh_ciphers|join(',') if ssh_ciphers else 'Ciphers'|comment }}
+{{ 'Ciphers ' ~ ssh_ciphers|join(',') if ssh_ciphers and ssh_ciphers_config else 'Ciphers'|comment }}
```

Comment:
This and the changes for MACs and Kex shouldn't be needed since `ssh_ciphers` can only be set in two ways:

* define it as a variable, then `ansible.builtin.include_tasks: crypto_ciphers.yml` won't be run.
* by the tasks in `ansible.builtin.include_tasks: crypto_ciphers.yml`

Or is there something I am missing where we want to define Ciphers but not actually configure them?

Reply:
Actually on main we have :

- If I don't define them as a variable, the tasks crypto_ciphers.yml will define them for me and the template will use the defaults and hardcode them in the configuration file.
- If I define them as a non true variable, the same apply as I wasn't defining them.
- If I define them as a variable, the template will use this variable and hardcode them in the configuration file.

So what about if I doesn't want to define Ciphers and does not want to configure them ?